packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js

```diff
@@ -80,7 +80,7 @@ export default class QuanticSearchInterface extends LightningElement {
   unsubscribeUrlManager;
 
   connectedCallback() {
-    loadDependencies(this).then((CoveoHeadless) => {
```

Comment:
The type checker was confused between `CoveoHeadless` (the module object), and `CoveoHeadless` (the return of `loadDependencies`).